deployment/src/docker/webapp.Dockerfile

```diff
@@ -13,8 +13,10 @@ ARG CONFIGURATION=example
 
 # copy public constants into the Docker image to brand it
 COPY configurations/${CONFIGURATION}/branding/static/ static/
-COPY configurations/${CONFIGURATION}/branding/constants/ constants/
-RUN /bin/sh -c 'cd constants && for f in *.ts; do mv -- "$f" "${f%.ts}.js"; done'
+# Wolle: COPY configurations/${CONFIGURATION}/branding/constants/ constants/
```

Comment:
```suggestion
```

Reply:
I's a work in progress comment. I'll remove it later.